apps/i18n/common/en_us.json

```diff
@@ -1917,5 +1917,8 @@
   "yourSchoolParentDesc": "Take action to expand computer science in your school or district",
   "yourSchoolTeacherButton": "Start teaching",
   "yourSchoolTeacherDesc": "Start teaching today, no computer science experience necessary",
-  "yourSchoolTellUs": "Tell us about your school"
+  "yourSchoolTellUs": "Tell us about your school",
+  "whyWithQuestionMark": "Why?",
+  "noSectionDialogHeader": "Why don't {classroom} sections have Section Codes?",
+  "noSectionDialogBody": "{classroom} sections' lists of students reflect your externally-managed {classroom} roster. In order to add or remove a student from this section, please first add or remove them from the corresponding classroom in {classroom}, then click the “Sync Students from {classroom}’ button in the top left corner of the Manage Students tab."
```

Comment:
Quick FYI: Smart quotes can cause issues in the i18n sync. Replace the “ and ’ here with " and '.

Reply:
✔️